packages/core/src/components/RadioButton/RadioButtonRow.tsx

```diff
@@ -22,7 +22,7 @@ export enum Direction {
 
 export interface RadioButtonRowProps extends Omit<RadioButtonProps, "onPress"> {
   label: string | React.ReactNode;
-  value: string; // A string that this radio button row represents when selected
+  value: string; // A string (or number that will be parsed String(number)) that this radio button row represents when selected
```

Comment:
If a number is passed in here (like in the example screen), shouldn't this be `value: string | number;`?

Reply:
@donald done.